### DISCUSSION TITLE
[New Game Asset Request] - Mushroom Kingdom Fusion and Super Mario Fusion: Revival

### DISCUSSION THREAD

**Discussion by fexhvhbdsf864** [Wohlstand/TheXTech#520] [Game projects]
This Two Games Used Game Maker Development Fan-game. can port to TheXTech Engine?
Thanks.

**Comment by Wohlstand** [comment]
Everything that was made on Game Maker is totally different from that TheXTech and other SMBX branches do. It's possible to "remake" rather than port. But, technically, is possible to make a script that will convert levels into TheXTech format, and there is a lot of work on making all logic work close to original.

**Comment by fexhvhbdsf864** [comment]
> Everything that was made on Game Maker is totally different from that TheXTech and other SMBX branches do. It's possible to "remake" rather than port. But, technically, is possible to make a script that will convert levels into TheXTech format, and there is a lot of work on making all logic work close to original.

OK,Thanks.I should says is remake not port.

**Comment by Wohlstand** [comment]
Anyway, I'll convert this issue into discussion. It's not just a task for the engine.

**Reply by fexhvhbdsf864** [reply]
So,When is start this project?
